document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -266,8 +266,18 @@ protected String parseNonCompositeFilter(Filter filter, Params.Builder paramsBui
         }
         break;
       case NEQ:
-        // Disabled for parity with mongo collection API.
-        // Can be implemented by wrapping predicate for EQ in a NOT()
+        sqlOperator = " != ";
+        // https://github.com/hypertrace/document-store/pull/20#discussion_r547101520
+        // The expected behaviour is to get all documents which either satisfy non equality condition
+        // or the key doesn't exist in them
+        // i.e. "document->key IS NULL OR document->key->> != value"
+        // Semantics for postgres operator in jsonb is of key EXISTS and NOT EQUALS value
```

Comment:
Can we replace this comment for the last two lines as below?
```
// Semantics for handling if key not exists and if it exists, its valued 
// doesn't for Jsonb document will be done as, 
// "document->key IS NULL OR document->key->> != value"
```

Reply:
done